docs/site/downloads/cldr-47.md

```diff
@@ -50,77 +49,91 @@ Count | Level | Usage | Examples
+* **Javascript:** [messageformat](https://github.com/messageformat/messageformat/tree/main/mf2/messageformat) 4.0 provides a formatter and conversion tools for the MessageFormat 2 syntax, together with a polyfill of the runtime API proposed for ECMA-402.
+
+(Because of the timing, these implement a slightly earlier version of the spec, but can be used for initial evaluation, testing, and experimentation.)
+
+## Specification Changes
+
+The following is a summary of the most significant changes to the specification (LDML) since CLDR 46.
+For more information, see [LDML 47 beta](https://unicode.org/reports/tr35/dev/tr35.html#modifications).
+
+- Many changes to MessageFormat, including:
+    - Now stable, with the stabiliy policy now normative.
+    - Added and clarified terminology
+    - Modified portions of the syntax (ABNF)
+    - Revised the Default Bidi Strategy, and added support for bidirectional isolates and marks
+    - Enabled functions to know whether an option value was set using a literal or a variable, which is necessary for some function's selection mechanism (see below).
+    - Updated the default functions
+        - Only three default functions are stable: :string, :number, and :integer. Other functions are Draft.
+- Added a Time Precision option to Semantic Datetime Skeletons, replacing discrete time fields
+- Significant changes in Timezones, Unit Identifiers, Keyboard Transforms, Currency elements, and others.
+- Moved @MATCH documentation to the site (it is for internal tooling)
 
 There are many more changes that are important to implementations, such as changes to certain identifier syntax and various algorithms.
 See the [Modifications section](https://www.unicode.org/reports/tr35/proposed.html#Modifications) of the specification for details.
 
 ## Data Changes
 
-**TBD: Flesh out overview items**
-   - Updated language matching for Afrikaans to English (en) from Dutch (nl) [CLDR-18198](https://unicode-org.atlassian.net/browse/CLDR-18198)
-   - Ordered scripts in `<languageData>` in descending order of usage per locale [CLDR-18155](https://unicode-org.atlassian.net/browse/CLDR-18155)
-   - Fixed certain invalid codes [CLDR-18129](https://unicode-org.atlassian.net/browse/CLDR-18129)
-
 ### DTD Changes
-
-Most of the DTD changes were in 46.1. One additional change was to order currency values in **TBD get ticket number**
+Changes include:
+- Ordering currencies used in each region
+- Adding script data (for example, relating Jpan to its component scripts)
```

Comment:
Should this say "Japan"?

Reply:
I think we just need to gloss the script name with the human readable version.

Reply:
Adding script data (for example, relating the script code Jpan to its component script codes)